clin/clients/nakadi_sql.py

```diff
@@ -68,6 +68,13 @@ def convert_repartitioning() -> Optional[Partitioning]:
             else None
         )
 
+    def get_compaction_key() -> Optional[str]:
+        return (
+            payload["partition_compaction_key_field"]
+            if "partition_compaction_key_field" in payload
+            else None
+        )
+
```

Comment:
You don't need that as `dict.get()` returns `None` if the key is not present in the dictionary